src/docfx/lib/UrlUtility.cs

```diff
@@ -348,6 +348,31 @@ public static string RemoveLeadingHostName(string url, string hostName, bool rem
                 : $"/{path}";
         }
 
+        public static string StandardizeBookmark(string uid)
+        {
+            uid = uid.ToLowerInvariant();
+
+            uid =
```

Comment:
so when we chain calls, the usual practice is to put `.` at the beginning:

```csharp
uid = uid
    .A
    .B
```

Reply:
Fixed